bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -416,14 +419,7 @@ protected ChannelFuture connect() {
             bootstrap.channel(NioSocketChannel.class);
         }
 
-        ByteBufAllocator allocator;
-        if (this.conf.isNettyUsePooledBuffers()) {
```

Comment:
isNettyUsePooledBuffers should be marked as deprecated now, and the deprecation note pointing to the new method.

Reply:
Sure